File: exercicios/R/1009/1009.r
input = file('stdin','r')

nome = readLines(input, n=1)
salario = as.double(readLines(input, n=1))
vendas = as.double(readLines(input, n=1))

total = (vendas * 0.15) + salario

write(sprintf("TOTAL = R$ %.2f",total),'')

File: exercicios/JavaScript/1009/1009.js
var input = require('fs').readFileSync('/dev/stdin','utf8');
var lines = input.split('\n');

var name = string(lines.shift());
var salario = parseFloat(lines.shift());
var totalvenda =  parseFloat(lines.shift());

var newSalary = salario + (totalvenda * 0.15);

console.log("TOTAL = R$ " + newSalary.toFixed(2));